pkg/controller/installation/core_controller.go

```diff
@@ -1214,9 +1214,16 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 	// If we're on OpenShift on AWS render a Job (and needed resources) to
 	// setup the security groups we need for IPIP, BGP, and Typha communication.
 	if openShiftOnAws {
+		// Detect if this cluster is an OpenShift HPC hosted cluster, as AWS
+		// security group setup is different in this case.
+		isHostedOpenShift, err := utils.IsHostedOpenShift(ctx, r.config)
+		if err != nil {
+			log.Info(err.Error())
```

Comment:
I think we want to return an error in this branch -  What happens if this fails and we attempt to continue anyway?